src/Index.tsx

```diff
@@ -50,7 +50,10 @@ if ($featureFlags.vendorEngrams) {
   saveVendorDropsToIndexedDB();
 }
 updateCSSVariables();
-store.dispatch(loadGlobalSettings());
+
+if ($featureFlags.dimApi) {
```

Comment:
I see `SyncService.init()` is called below (~L61) - should we consider moving it to the else block of this statement so we don't load the gapi for users who don't need it?

Reply:
Or if not the else statement, in the init() function of the SyncService, to return early if the dimAPI has loaded the config previously?

Reply:
Daaah, I see we're continuing to sync to google drive while we're in the beta. Sounds good!

Reply:
Yeah originally I'd wanted it to be a bit more aggressive but backed off for now.